backend/business-partner-agent/src/main/java/org/hyperledger/bpa/controller/UserController.java

```diff
@@ -95,31 +90,6 @@ private Map<String, Object> buildModel(@NonNull Boolean enabled) {
         return model;
     }
 
-    @Secured({ "ROLE_ADMIN" })
-    @Post(value = "/register", consumes = { MediaType.APPLICATION_JSON, MediaType.APPLICATION_FORM_URLENCODED })
```

Comment:
That one is debatable, it is not exposed in the UI, but potentially someone could use it. What I don't like though is that this can be used to create a lot of users, as there is no user management no one will notice it, so this is a potential security risk, especially when used together with Keycloak as both mechanisms can coexist. So I really really want to remove all code related the user management asap from the BPA on favour of oauth.